modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,31 @@
 = What’s New in Studio
+
+Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
+
+This version also improves the metadata edition and Design Center import experiences and adds a feature to comment and uncomment objects from the canvas.
+
+== Comprehensive Publication to Exchange
+
+When publishing an asset to Exchange, you can now add more information such as tags, categories, name, description, and documentation to your asset directly from Studio.
+
+== MFA Support
+
+You can now log in to Anypoint Platform using multi-factor authentication (MFA) from the Studio built-in browser.
+
+== Comment and Uncomment Objects from the Canvas
```

Comment:
```suggestion
== Comment and Uncomment Objects Directly from the Canvas
```
I kept struggling with whether the user was taking action *on* the canvas or *from* the canvas, but I chose to stick with your original wording.